packages/marketing-analytics-browser/src/gtm-snippet-index.ts

```diff
@@ -0,0 +1,44 @@
+import { getGlobalScope } from '@amplitude/analytics-client-common';
```

Comment:
I find even with that, it's a little bit hard because in 26-28, we assign the variable to another object.  
So we have the logic like this 
 ```let GlobalAmplitude = isGTM ? GlobalScope.amplitudeGTM : GlobalScope.amplitude;```
Then later we cannot assign   `GlobalAmplitude` to either 
`Object.assign(GlobalScope.amplitudeGTM || {}, amplitudeGTM, {
    createInstance: createNamedInstance,
  });` or 
`Object.assign(GlobalScope.amplitude || {}, amplitude, {
    createInstance: createNamedInstance,
  });`
I will leave it as it is at this time.